front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -225,8 +236,24 @@ export const usePathfinding = (
 
         try {
           const pathfindingResult = await postPathfindingBlocks(pathfindingInput).unwrap();
+          if (pathfindingResult.status === 'invalid_path_items') {
+            const invalidPathItems = pathfindingResult.items
+              .map((item) => {
+                return 'trigram' in item.path_item ? item.path_item.trigram : null;
+              })
+              .filter((trigram): trigram is string => trigram !== null);
 
-          if (
+            if (invalidPathItems.length > 0) {
+              const updatedPathSteps = pathSteps.map((step) => {
+                if (step && 'trigram' in step && invalidPathItems.includes(step.trigram)) {
+                  return { ...step, invalid: true };
+                }
+                return step;
+              });
+              dispatch(updatePathSteps({ pathSteps: updatedPathSteps }));
+              pathfindingDispatch({ type: 'PATHFINDING_PARAM_CHANGED' });
+            }
```

Comment:
Maybe you can dispatch here a message for the user which explains that the pathfinding has failed because some pathSteps were not found on the infra ?
We should display this message even if invalidPathSteps is empty (as it contains only the invalidPathSteps with trigram and not all the invalidPathSteps)

Reply:
Now even if there is an invalid pathSteps, the pathfinding succeed (if there is at least two valid pathSteps).
Do we still want to dispatch a message then ?